client/components/happychat/agent-w/index.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+function rand( min, max ) {
+	return Math.floor( Math.random() * ( max - min + 1 ) ) + min;
+}
+
+export const emotions = [ 'blush', 'wink', 'dreamy', 'somber', 'asleep', 'sad', 'cool', 'electric' ];
+
+export function RandomAgentW( { exclude = [] } ) {
+	const filteredEmotions = emotions.filter( emotion => exclude.indexOf( emotion ) === -1 );
+	const index = rand( 0, filteredEmotions.length - 1 );
+
+	return (
+		<AgentW classes={ filteredEmotions[ index ] } />
+	)
+}
+
+const AgentW = ( props ) => {
+	return (
+		<div className="chat__gravatar chat__agent-w">
```

Comment:
eslint flags a bunch of stuff for agent-w (like not following CSS naming conventions) but we'll hopefully move it to a shared module soon and get around that problem :)